### PR TITLE
New Picture of login Screen

### PR DESCRIPTION
**Pull request by alihusnainarshad**
Image of Email code authentication method input during WordPress login

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -4,23 +4,21 @@
...
 **Tags:** [two factor](https://wordpress.org/plugins/tags/two-factor), [two step](https://wordpress.org/plugins/tags/two-step), [authentication](https://wordpress.org/plugins/tags/authentication), [login](https://wordpress.org/plugins/tags/login), [totp](https://wordpress.org/plugins/tags/totp), [fido u2f](https://wordpress.org/plugins/tags/fido-u2f), [u2f](https://wordpress.org/plugins/tags/u2f), [email](https://wordpress.org/plugins/tags/email), [backup codes](https://wordpress.org/plugins/tags/backup-codes), [2fa](https://wordpress.org/plugins/tags/2fa), [yubikey](https://wordpress.org/plugins/tags/yubikey)  
 **Requires at least:** 4.3  
 **Tested up to:** 5.2  
 **Stable tag:** trunk (master)  
 
-[![Build Status](https://travis-ci.org/georgestephanis/two-factor.svg?branch=master)](https://travis-ci.org/georgestephanis/two-factor) [![Coverage Status](https://coveralls.io/repos/georgestephanis/two-factor/badge.svg?branch=master)](https://coveralls.io/github/georgestephanis/two-factor) [![Built with Grunt](https://gruntjs.com/cdn/builtwith.svg)](http://gruntjs.com)
```
Comment (kasparsd) [inline readme.md]: This `readme.md` file is generated automatically from the `readme.txt` using the `npm run readme` command:

https://github.com/WordPress/two-factor/blob/ab8d6510a17606b200d1e44a603c13b5014bac52/package.json#L9

Could you please make the changes only to the `readme.txt` file and run `npm run readme` to update this file?

Sorry this hasn't been documented anywhere.

File: readme.txt [hunk 2]
```diff
@@ -8,23 +8,24 @@ Stable tag: trunk
...
-
-Use the "Two-Factor Options" section under "Users" → "Your Profile" to enable and configure one or multiple two-factor authentication providers for your account:
-
-- Email codes
-- Time Based One-Time Passwords (TOTP)
-- FIDO Universal 2nd Factor (U2F)
-- Backup Codes
-- Dummy Method (only for testing purposes)
+
+Use the "Two-Factor Options" section under "Users" → "Your Profile" to enable and configure one or multiple two-factor authentication providers for your account:
```
Comment (kasparsd) [inline readme.txt]: What are the changes in this section of the diff? Is that the line endings or extra whitespace? Those should match the definitions in `.editorconfig`:

https://github.com/WordPress/two-factor/blob/ab8d6510a17606b200d1e44a603c13b5014bac52/.editorconfig

File: readme.txt [hunk 3]
```diff
@@ -8,23 +8,24 @@ Stable tag: trunk
...
-1. Two-factor options under User Profile.
-2. U2F Security Keys section under User Profile.
-
+
+
+== Screenshots ==
+
+1. Two-factor options under User Profile.
+2. U2F Security Keys section under User Profile.
+3. Email code authentication method input during WordPress login
```
Comment (kasparsd) [inline readme.txt]: Could we end this new item with a period to match the rest of the items?

**Comment by kasparsd** [comment]
Could you please upload a full size screenshot without the resizing artifacts? Maybe that's only on my screen so I'm not sure.

screenshot-3

And maybe run it through a lossless image compression tool that could optimize the file size?

compressed

**Comment by alihusnainarshad** [comment]
I have submitted a new PR thats why cosing this PR
Thank you very much @kasparsd for the instructions

**Comment by kasparsd** [comment]
@alihusnainarshad Making changes to the same branch and pull request allows to keep a history of the full changeset and the changes included and discussed. Could you please push the fixes to this same branch?

**Pull request closed without merging**